src/test/java/net/sf/jabref/importer/fileformat/BibTeXMLImporterTestFiles.java

```diff
@@ -0,0 +1,76 @@
+package net.sf.jabref.importer.fileformat;
+
+import net.sf.jabref.*;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+@RunWith(Parameterized.class)
+public class BibTeXMLImporterTestFiles {
+
+    private static final Pattern PATTERN = Pattern.compile("\\D*[0123456789]");
+
+    private BibTeXMLImporter bibtexmlImporter;
+
+    @Parameter
+    public String fileName;
+
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
+        bibtexmlImporter = new BibTeXMLImporter();
+    }
+
+    @Parameters(name = "{0}")
+    public static Collection<String> fileNames() {
+        List<String> files = new ArrayList<>();
+        File d = new File(System.getProperty("user.dir") + "/src/test/resources/net/sf/jabref/importer/fileformat");
+        for (File f : d.listFiles()) {
+            files.add(f.getName());
+        }
+        return files.stream().filter(n -> n.startsWith("BibTeXMLImporterTest")).collect(Collectors.toList());
+    }
+
+    @Ignore("Because the importer needs to be reimplemented")
+    @Test
+    public void testIsRecognizedFormat() throws IOException {
+        try (InputStream stream = BibTeXMLImporterTest.class.getResourceAsStream(fileName)) {
+            Assert.assertTrue(bibtexmlImporter.isRecognizedFormat(stream));
+        }
+    }
+
+    @Ignore("Because the importer needs to be reimplemented")

```

Comment:
Is it really true that all test files are not working? Isn't there a single file working? All files carry `bibtex:` as prefix. Please elaborate more why the tests are ignored. Or, better, add working files only and keep the non-working files at your fork. We should really focus to keep the JabRef codebase clean. It might happen that the results of the discussion at #938 reveals, that we will drop BibTeXML support.